arm9/src/unk_02088DD8.c

```diff
@@ -0,0 +1,76 @@
+#include "global.h"
+#include "heap.h"
+#include "pokemon.h"
+#include "unk_02088DD8.h"
+
+THUMB_FUNC struct UnkStruct_02088DD8* FUN_02088DD8(u32 heap_id) {
+    struct UnkStruct_02088DD8 *returnPointer = AllocFromHeap(heap_id, sizeof(struct UnkStruct_02088DD8));
+    __builtin__clear(returnPointer, sizeof(struct UnkStruct_02088DD8));
+    return returnPointer;
+}
+
+THUMB_FUNC void FUN_02088DF0(struct UnkStruct_02037CF0 *r0) {
+    FreeToHeap(r0);
+}
+
+#define WOTBL_END           0xFFFF
+#define WOTBL_MOVE_MASK     0x01FF
+#define WOTBL_MOVE_SHIFT         0
+#define WOTBL_LVL_MASK      0xFE00
+#define WOTBL_LVL_SHIFT          9
+#define WOTBL_MOVE(x) ((u16)(((x) & WOTBL_MOVE_MASK) >> WOTBL_MOVE_SHIFT))
+#define WOTBL_LVL(x) (/*(u8)*/(((x) & WOTBL_LVL_MASK) >> WOTBL_LVL_SHIFT))
+// i don't know why either.
```

Comment:
???

Reply:
The code matches WOTBL_LVL as previously defined, except that the previously defined WOTBL_LVL casts to u8 but the code in this file does not.